main/src/com/google/refine/importers/XmlImporter.java

```diff
@@ -209,7 +209,9 @@ public void parseOneFile(Project project, ProjectMetadata metadata,
     
     static public class XmlParser implements TreeReader {
         final protected XMLStreamReader parser;
-        
+        static final String WHITESPACES_PATTERN = "^[\\n\\r\\t].*$";
```

Comment:
Instead of introducing our own regex here, perhaps we could use `CharMatcher.whitespace()` just like in other places in the code (see for instance `main/src/com/google/refine/expr/functions/strings/Trim.java`).

Reply:
@wetneb Thanks for the suggestion. I have updated the PR with new changes. Please let me know if anything else we can enhance.